types/tape-async/tslint.json

```diff
@@ -0,0 +1,83 @@
+{
+	"extends": "dtslint/dt.json",
+	"rules": {
```

Comment:
Please remove these overrides so that the linter runs with default settings. You might have to change some minor things, but from reading the code, I doubt it will be much.

Reply:
I have kept the `indent` rule, as it doesn’t have a default, and I use tabs in this type definition.

I have also kept the `prefer‑const: false` rule, as `const` properties must be initialised.

I’ve also added the `dt‑header: false` rule, since the `// TypeScript Version: 2.x` comments in the tests cause it to fail.

Reply:
`dt-header` is pretty important. 

1. What error do the tests give?
2. The library requires 2.3, so there's no way you can use it with 2.1, so I don't think there's any point

The other rules are not that important, but there's value in having a tslint.json with no overrides, because we run scripts over Definitely Typed occasionally, and those scripts often have to skip packages with overrides since they don't follow the normal rules. So I'd prefer to have all 3 overrides gone.

The usual way to get around `prefer-const` is to make your unitialised variables into parameters instead:

```ts
let x: number;
g(x)
// =>
function f(x: number) {
  g(x)
}
```

Reply:
`dt-header` complains that the&nbsp;tests contain&nbsp;`// TypeScript Version: 2.x`, like&nbsp;I&nbsp;said&nbsp;in:&nbsp;#33491:

```txt
ERROR: …/DefinitelyTyped/types/tape-async/test/tape-async.async.test.ts:1:1 - TypeScript version should be specified under header in `index.d.ts`. See: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md
ERROR: …/DefinitelyTyped/types/tape-async/test/tape-async.generators.test.ts:1:1 - TypeScript version should be specified under header in `index.d.ts`. See: https://github.com/Microsoft/dtslint/blob/master/docs/dt-header.md
```